.circleci/config.yml

```diff
@@ -266,57 +268,101 @@ jobs:
             docker login --username "${DOCKER_USER}" --password "${DOCKER_PASSWORD}"
             ./gradlew --no-daemon --parallel "-Pbranch=${CIRCLE_BRANCH}" dockerUpload
 
+  publishAPIDoc:
+    executor: medium_executor
+    steps:
+      - prepare
+      - add_ssh_keys:
+          fingerprints:
+            - '45:36:58:67:43:15:c6:5f:2c:58:ec:7f:71:e2:e6:ef'
+      - run:
+          name: Build runnable Teku binaries
+          command: |
+            ./gradlew --no-daemon --parallel installDist
+      - run:
+          name: Extract OpenAPI JSON
+          command: |
+            build/install/teku/bin/teku --rest-api-docs-enabled=true &
+            TEKU_PID=$!
+            wget -O api.json http://localhost:5051/swagger-docs
```

Comment:
Isn't there a race condition here? We'd need to wait until the rest api service has actually started listening.

Reply:
wget does some retries.